apps/meteor/app/lib/server/startup/settings.ts

```diff
@@ -14,6 +14,13 @@ settingsRegistry.add('Initial_Channel_Created', false, {
 	hidden: true,
 });
 
+// this settings will let clients know in case presence has been disabled
+settingsRegistry.add('Presence_broadcast_disabled', false, {
+	type: 'boolean',
+	public: true,
+	// hidden: true,
```

Comment:
Why the setting is not hidden anymore @ggazzo ?

Reply:
even being public, hidden settings are not sent to client =/